v22.1/restore.md

```diff
@@ -340,6 +340,52 @@ RESTORE DATABASE bank FROM LATEST IN 's3://{bucket_name}?AWS_ACCESS_KEY_ID={key_
  `RESTORE` will re-validate [indexes](indexes.html) when [incremental backups](take-full-and-incremental-backups.html) are created from an older version (v20.2.2 and earlier or v20.1.4 and earlier), but restored by a newer version (v21.1.0+). These earlier releases may have included incomplete data for indexes that were in the process of being created.
 {{site.data.alerts.end}}
 
+### Restore with `AS OF SYSTEM TIME`
+
+If you did **not** run a backup with `revision_history`, it is still possible to use `AS OF SYSTEM TIME` with `RESTORE` to target a particular time for the restore. However, your restore will be limited to the times of the full backup and each incremental backup in the chain. 
+
+This is different to a backup with revision history that captures every change made within the garbage collection period leading up to and including the given timestamp. See [Take Backups with Revision History and Restore from a Point-in-time](take-backups-with-revision-history-and-restore-from-a-point-in-time.html) for more detail.
+
+In the case where there is no backed up revision history, use the following example to restore to a particular time.
+
+First, find the times that are available for a point-in-time-restore by listing the available backup directories in your storage location:
+
+{% include_cached copy-clipboard.html %}
+~~~sql
+SHOW BACKUPS IN 's3://{bucket_name}?AWS_ACCESS_KEY_ID={key_id}&AWS_SECRET_ACCESS_KEY={access_key}';
+~~~
+~~~
+          path
+------------------------
+  2023/01/18-141753.98
+  2023/01/23-184816.10
+  2023/01/23-185448.11
+(3 rows)
+~~~
+
+From the output use the required date directory and run the following to get the details of the backup:
```

Comment:
Small language suggestion:

> Next, provide the desired `path` from the output to the `SHOW BACKUP` command to get the details of the backup:

I matched `path` to the command output, but feel free to stick with `date directory` if you prefer, or if you use it extensively elsewhere. Up to you.

Reply:
Honestly, this is probably a mix... I'm going to keep it as date directory for now as I feel like it is useful for this specific time-based example.